File: src/01-easy/horse-racing-duals/solution.cs
using System;
using System.Linq;
class Solution {
    static void Main() {
        int n = int.Parse(Console.ReadLine());
        int[] h = new int[n];
        for (int i = 0; i < n; i++) h[i] = int.Parse(Console.ReadLine());
        Array.Sort(h);
        int min = h[1] - h[0];
        for (int i = 1; i < n - 1; i++) min = Math.Min(min, h[i+1] - h[i]);
        Console.WriteLine(min);
    }
}

File: src/01-easy/horse-racing-duals/solution.vb
Module Solution
    Sub Main()
        Dim n As Integer = CInt(Console.ReadLine())
        Dim h(n - 1) As Integer
        For i As Integer = 0 To n - 1
            h(i) = CInt(Console.ReadLine())
        Next
        Array.Sort(h)
        Dim m As Integer = h(1) - h(0)
        For i As Integer = 1 To n - 2
            Dim d As Integer = h(i + 1) - h(i)
            If d < m Then m = d
        Next
        Console.WriteLine(m)
    End Sub
End Module
